### ISSUE TITLE
Query about Persistent Use of Masking during Inference and Finetuning in MAE

### ISSUE THREAD

**Issue by rlgnswk** [IcarusWizard/MAE#23]
Hello,

I have been reviewing the MAE implementation, and I noticed that during both inference and finetuning, the model continues to use the same mask ratio (0.75) as was used during pretraining. Could you clarify why the model does not encode the entire image instead of using a masking approach? I am curious about the advantages or the rationale behind continuing with this masking strategy post-pretraining.

Thank you for your insights!

**Comment by IcarusWizard** [comment]
Hi, in finetuning, the encoder is reloaded to [ViT_Classifier](https://github.com/IcarusWizard/MAE/blob/11c21fcbdb4d26a7ef84cf4d95679921c958105a/model.py#L145) where the masking is discarded.

**Comment by rlgnswk** [comment]
Thank you for your reply!
Is this way same with the original MAE?

**Comment by IcarusWizard** [comment]
Yes, that is the same with the original MAE, although their implementation may be slightly different.

**Comment by rlgnswk** [comment]
Thanks a lot for your kindness. 
However, I can not find where the masking function off. 
Could you indicate where is it?

**Comment by IcarusWizard** [comment]
[Here](https://github.com/IcarusWizard/MAE/blob/11c21fcbdb4d26a7ef84cf4d95679921c958105a/train_classifier.py#L47) we use the encoder from MAE to build the [ViT_Classifier](https://github.com/IcarusWizard/MAE/blob/11c21fcbdb4d26a7ef84cf4d95679921c958105a/model.py#L145). In the `__init__` and `forward` [here](https://github.com/IcarusWizard/MAE/blob/11c21fcbdb4d26a7ef84cf4d95679921c958105a/model.py#L145-L164) you can see anything except the masking is in use.

**Comment by rlgnswk** [comment]
I really appreciate it !
